### ISSUE TITLE
monitor / display wifi and lora signal strength

### ISSUE THREAD

**Issue by ThomasWaldmann** [ecocurious2/MultiGeiger#307]
this might be useful for positioning / directing the multigeiger.

problem: OLED is not visible when putting the whole thing into the grey plastic tube.

so only these are left to display status:
- wifi and builtin web interface
- usb log output

**Comment by ThomasWaldmann** [comment]
Hmm, is this technically possible?
- We can only access the web status page when the AP is active.
- But we can only know WiFi signal strength while the wifi client is active.

**Comment by rexfue** [comment]
No, we can see the web page at every time: point your browser to the ip address of the geiger ( or to http:// ESP32-xxxx with xxx the chip id)

**Comment by ThomasWaldmann** [comment]
Ah, right, but that's going over the wifi client connection then and you can't rely on this if you have:
- connection troubles to uplink wifi AP
- no wifi uplink, only LoRa

**Comment by t-pi** [comment]
Hmm, a fully blown config interface via BLE is indeed some time away, but some additional info might be relatively easy to implement. Which info would be required, and is it already available?

**Comment by ThomasWaldmann** [comment]
Maybe keep the BLE stuff to existing functionality for now. This ticket is about not yet existing stuff. :-)

See also #308.

**Comment by M-a-r-i-a-n-n-e** [comment]
@ThomasWaldmann I would prefer the same solution in grafana as for all sensor.community devices: https://api-rrd.madavi.de/grafana/d/Fk6mw1WGz/wifi-signal?orgId=1&var-chipID=esp8266-4206630 (I just selected an arbitrary ESP).
